docs/apache-airflow/security/secrets/secrets-backend/index.rst

```diff
@@ -38,8 +38,16 @@ Search path
 When looking up a connection/variable, by default Airflow will search environment variables first and metastore
 database second.
 
-If you enable an alternative secrets backend, it will be searched first, followed by environment variables,
-then metastore.  This search ordering is not configurable.
+.. versionchanged:: 2.3.1
```

Comment:
Should be 2.4.0. New features are added to the next minor release

Reply:
No problem I will change to 2.4.0. 

I confused that some of the features added in the path versions (2.2.4, 2.2.5, etc) however I didn't investigate much, probably it was some bug fixes